app/views/admin/publishers/show.html.slim

```diff
@@ -18,6 +18,9 @@
         .db-info-row
           .db-field = "Uphold connected:"
```

Comment:
could we change the text to 'uphold_verified' to match the database? both 'uphold_verified' and 'uphold connected' don't really give a full picture, but at least 'uphold_verified' matches the database.  we also use the term 'uphold connected' in place of 'uphold authorized' when we display to the pub.

Reply:
Updated the nomenclature 🌮 
https://github.com/brave-intl/publishers/pull/1366/commits/43ebb4cfc125e9e6f5052b4f8edfdcb8ee21b8f3